<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizWindow</class>
 <widget class="QDialog" name="QuizWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 63, 80)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="pageHome">
      <widget class="QPushButton" name="btnTakeQuiz">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>170</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Quiz</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSetting">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>250</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255); </string>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLogout">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>330</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>LogOut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>QUIZ APP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageSetting">
      <widget class="QPushButton" name="btnChgFont">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Change Font</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnChgBgColor">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>270</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Change Background Color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>350</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>SETTING PAGE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageQuizSubjects">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>SELECT SUBJECT FOR QUIZ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBiology">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>150</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Biology</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnComputing">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>210</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Computing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCustom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>270</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGeography">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>330</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Geography</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>480</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageQuiz">
      <widget class="QLabel" name="lblQuizTitle">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>QUIZ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollAreaQuiz">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>521</width>
         <height>391</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>519</width>
          <height>389</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnBack_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>520</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCheckAnswers">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>520</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Check Answers</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
